tests/hipscat_import/catalog/test_run_import.py

```diff
@@ -135,6 +134,98 @@ def test_resume_dask_runner(
+    tmp_path,
+):
+    """Test execution in the presence of resume files that are not compatible
+    with the new set of running arguments."""
+    ## First, copy over our intermediate files.
+    ## This prevents overwriting source-controlled resume files.
+    intermediate_dir = os.path.join(tmp_path, "resume_catalog", "intermediate")
+    shutil.copytree(
+        os.path.join(resume_dir, "intermediate"),
+        intermediate_dir,
+    )
+
+    ## Now set up our resume files to match previous work.
+    resume_tmp = os.path.join(tmp_path, "tmp", "resume_catalog")
+    ResumePlan(tmp_path=resume_tmp, progress_bar=False)
+    histogram = hist.empty_histogram(0)
+    histogram[11] = 131
+    empty = hist.empty_histogram(0)
+    for file_index in range(0, 5):
+        ResumePlan.touch_key_done_file(resume_tmp, ResumePlan.SPLITTING_STAGE, f"split_{file_index}")
+        ResumePlan.write_partial_histogram(
+            tmp_path=resume_tmp,
+            mapping_key=f"map_{file_index}",
+            histogram=histogram if file_index == 0 else empty,
+        )
+
+    ResumePlan.touch_key_done_file(resume_tmp, ResumePlan.REDUCING_STAGE, "0_11")
+
+    shutil.copytree(
+        os.path.join(resume_dir, "Norder=0"),
+        os.path.join(tmp_path, "resume_catalog", "Norder=0"),
+    )
+
+    with pytest.raises(ValueError, match="incompatible with the current healpix order"):
+        args = ImportArguments(
+            output_artifact_name="resume_catalog",
+            input_path=small_sky_parts_dir,
+            file_reader="csv",
+            output_path=tmp_path,
+            dask_tmp=tmp_path,
+            tmp_dir=tmp_path,
+            resume_tmp=os.path.join(tmp_path, "tmp"),
+            lowest_healpix_order=1,
+            highest_healpix_order=1,
+            pixel_threshold=1000,
+            progress_bar=False,
+        )
+        runner.run(args, dask_client)
+
+    # Running with resume set to "False" will start the pipeline from scratch
+    args = ImportArguments(
+        output_artifact_name="resume_catalog",
+        input_path=small_sky_parts_dir,
+        file_reader="csv",
+        output_path=tmp_path,
+        dask_tmp=tmp_path,
+        tmp_dir=tmp_path,
+        resume_tmp=os.path.join(tmp_path, "tmp"),
+        lowest_healpix_order=1,
+        highest_healpix_order=1,
+        pixel_threshold=1000,
+        progress_bar=False,
+        resume=False,
+    )
+    runner.run(args, dask_client)
+
+    # Check that the catalog metadata file exists
+    catalog = Catalog.read_from_hipscat(args.catalog_path)
+    assert catalog.on_disk
+    assert catalog.catalog_path == args.catalog_path
+    assert catalog.catalog_info.ra_column == "ra"
+    assert catalog.catalog_info.dec_column == "dec"
+    assert catalog.catalog_info.total_rows == 131
+    assert len(catalog.get_healpix_pixels()) == 4
+
+    # Check that the catalog parquet files exist and
+    # that, collectively, they have the correct set of ids
+    ids = []
```

Comment:
we've got a fixture method that does all of this logic: `assert_parquet_file_ids`

Reply:
I'm generating the catalog at order 1 so I can't call this method just once but I did some polishing!